src/Maker/MakeController.php

```diff
@@ -60,19 +62,34 @@ public function getParameters(InputInterface $input): array
         $controllerClassName = Str::asClassName($input->getArgument('controller-class'), 'Controller');
         Validator::validateClassName($controllerClassName);
 
+        $title = "Hello {$controllerClassName}!";
+        if (file_exists($this->projectDir.'/templates/base.html.twig')) {
+            $twigFirstLine = "{% extends 'base.html.twig' %}\n{% block title %}{$title}{% endblock %}";
+        } else {
+            $twigFirstLine = "<!DOCTYPE html>\n<title>{$title}</title>";
+        }
+
         return [
             'controller_class_name' => $controllerClassName,
             'route_path' => Str::asRoutePath(str_replace('Controller', '', $controllerClassName)),
             'route_name' => Str::asRouteName(str_replace('Controller', '', $controllerClassName)),
+            'twig_file' => Str::asFilePath(str_replace('Controller', '', $controllerClassName)).'.html.twig',
+            'twig_first_line' => $twigFirstLine,
         ];
     }
 
     public function getFiles(array $params): array
     {
-        $skeletonFile = $this->isTwigInstalled() ? 'ControllerWithTwig.tpl.php' : 'Controller.tpl.php';
+        $dir = __DIR__.'/../Resources/skeleton/controller/';
+        $classFile = 'src/Controller/'.$params['controller_class_name'].'.php';
+
+        if (!$this->isTwigInstalled()) {
+            return [$dir.'Controller.tpl.php' => $classFile];
+        }
```

Comment:
I'm being annoying, but I'd rather see this reversed: the array with one entry only created first (and set to some `$paths` variable. Then an if statement to append the `.twig.php` file `if ($this->isTwigInstalled())`